change id to userinfo for more info, edited message commands.

AutumnClove opened the pull request:
Please make sure these boxes are checked (put an 'x' inside them) **before** creating the pull request.

- [x] I have tested my changes against the `review` branch (the latest developmental version), and this pull request is targeting that branch as a base
- [x] I have tested my changes on Python 3.5/3.6

----

### Description
ID is so boring why not provide some better info


### Related issues (if applicable)

AutumnClove committed:
change id to user for more info.

AutumnClove committed:
changed to return response 

changed to return response to follow current code set up a little better.

AutumnClove committed:
indentation

AutumnClove committed:
oops

tabs, not spaces 🤦

AutumnClove committed:
plz be fixed

AutumnClove committed:
.

MattBSG commented on musicbot/bot.py:
```diff
@@ -1317,18 +1317,33 @@ def _gen_embed(self):
+        creation_date = member.created_at
+        avatar = member.avatar_url if member.avatar_url else member.default_avatar_url
+        user_id = member.id
+        return Response(self.str.get('cmd-user', """
```
Please commit the updated `en.json` i18n file with the added string this references

AutumnClove committed:
did I do this right?

plz work lol

BabyBoySnow committed:
freaking commas

BabyBoySnow committed:
Experimental non mentions

Allows usage of calling users without having to mention them.  Has to use member name not nickname.

BabyBoySnow committed:
white spacing

ImthePeach reviewed (commented):
This should have a embed availability as well for cleaner look, user info never looks good when it's not in an embed. Command should be called `info` or `userinfo` instead of `user` for sanity reasons. You aren't asking for what user is this, you are asking for info on the user.

BabyBoySnow committed:
Sanity re-name

BabyBoySnow committed:
BEHOLD EMBED

BabyBoySnow committed:
in server

Some reason member.server doesn't work but member.server.name does so it's back now woo-hoo. Just a minor detail, also removes my rage comment of server not working in embeds whoops.

BabyBoySnow committed:
user => userinfo

Forgot to change that.
Due to the nature of the embed I can't really add it to i18n I think, however if anyone is willing to help feel free to commit and give some advice.

BabyBoySnow committed:
Nickname support

Use server.get_member_named to be able to look up through nicknames.

BabyBoySnow committed:
Edited messages

Allows support of edited messages commands. I don't know if this will be accepted I'll revert if not.

BabyBoySnow committed:
Re-ordering

Re-order to be more well ordered. Aesthetically pleasing.

BabyBoySnow committed:
Delete

Delete invoke let's be honest this is the only good option.

BabyBoySnow committed:
Embed color

Embed color is now members highest role color.

BabyBoySnow committed:
forgot to update non embed

Update the non embed version to be a little prettier looking, and follow the same order as the embed. Add check for delete messages since some people do like the commands called to not disappear I guess.

AutumnClove commented on musicbot/bot.py:
```diff
@@ -1317,18 +1318,66 @@ def _gen_embed(self):
-            return Response(self.str.get('cmd-id-self', 'Your ID is `{0}`').format(author.id), reply=True, delete_after=35)
+            stripped = len(self.config.command_prefix)+9
+            int(stripped)
+            ph = message.content[stripped:]
```
ph stands for placeholder, not pornhub LUL
